packages/design-system/src/components/utilities/margin.stories.tsx

```diff
@@ -12,7 +12,7 @@ export default {
 
 // an example for responsive margin -- referenced in doc site
 export const margin = () => (
-  <div className="ds-u-fill--primary-alt" style={{ maxWidth: 'fit-content' }}>
+  <div className="ds-u-fill--secondary-lighter" style={{ maxWidth: 'fit-content' }}>
```

Comment:
Maybe we should use `info` here, because we tend to use the info color for these utilities. For example, [on the current margin page](https://design.cms.gov/utilities/margin/?theme=core) the color is what is now `info`

Reply:
I think this actually highlights the need for a `ds-u-fill-info*` series of utility classes. We have all the other status colors.

Reply:
Merged that change

```suggestion
  <div className="ds-u-fill--info-lighter" style={{ maxWidth: 'fit-content' }}>
```